src/main/play-doc/operation/Install.md

```diff
@@ -233,7 +233,11 @@ _Perform each step in this section on all nodes: `172.17.0.1`, `172.17.0.2` and
 
 Proxying application endpoints is required when running more than one instance of ConductR; which should be always for production style scenarios. Proxying endpoints permits connectivity from both external callers and for bundle components to communicate with other bundle components. This also allows an external caller to contact an application that is running on any ConductR node by contacting any proxy instance.
 
-We will be using `HAProxy`. Install HAProxy version 1.5 or newer.
+We will be using `HAProxy` version 1.5 or newer.
+
```

Comment:
It seems I have the choice of installing HAProxy Docker or Native. If so, why might I prefer one v the other and which is recommended.